flocker/node/_deploy.py

```diff
@@ -131,9 +131,12 @@ def calculate_changes(self, configuration, cluster_state):
             local_state, configuration, cluster_state)
 
 
+def _eliot_system(part):
+    return u"flocker:p2pdeployer:" + part
```

Comment:
Not really accurate anymore (some of this is container stuff) but I guess we can fix that if/when we move container stuff and p2p stuff into their own modules.